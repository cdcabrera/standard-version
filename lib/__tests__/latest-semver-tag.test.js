const latestSemverTag = require('../latest-semver-tag');

jest.mock('git-semver-tags', () => (options, callback) => {
  const tags = [];
  callback(undefined, tags);
});

describe('latestSemverTag', () => {
  it('should attempt to get and parse tags', async () => {
    const result = await latestSemverTag();
    expect(result).toMatchSnapshot('default');
  });
});
