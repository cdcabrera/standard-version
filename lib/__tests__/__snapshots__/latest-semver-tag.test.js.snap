// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`latestSemverTag should attempt to get and parse tags: default 1`] = `"1.0.0"`;
